sycl/doc/XilinxFPGACompilation.md

```diff
@@ -31,6 +31,36 @@ Installing Xilinx FPGA compatible software stack:
   2. Xilinx runtime (XRT) for FPGAs: Download, build and install [XRT](https://github.com/Xilinx/XRT), this contains the OpenCL runtime.
   3. Xilinx SDx (2018.3+): Download and Install [SDx](https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/sdx-development-environments.html) which contains the `xocc` compiler.
 
+## Platforms
+
+It's of note that the SDx 2018.3 install comes with several platforms that do
```

Comment:
I have the feeling we should skip all this Zynq details and just focus on the Alveo boards, which are the OpenCL-supported boards.